README.md

```diff
@@ -18,7 +18,7 @@ Spike supports the following RISC-V ISA features:
   - D extension, v2.2
   - Q extension, v2.2
   - C extension, v2.0
-  - V extension, v0.7.1 (_requires a 64-bit host_)
+  - V extension, v0.8-draft-20191004 (_requires a 64-bit host_)
```

Comment:
I think we should also clarify that we support `Zvlsseg` but not `Zvamo` and `Zvediv` sub-extensions.

Reply:
agree and will update this part.